### ISSUE TITLE
thcrap_configure stuck loading with Wine

### ISSUE THREAD

**Issue by McUberStein** [thpatch/thcrap#178]
Every time I launch thcrap_configure through Wine and select the lang_en patch it gets stuck. Here is a screenshot: https://i.imgur.com/xWZfZHu.png , and this it the terminal output:  
                                             
01b0:fixme:heap:RtlSetHeapInformation 00000000 1 00000000 0 stub
01b0:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
00fc:fixme:imm:ImeSetActiveContext (0x44c2b0, 1): stub
00fc:fixme:imm:ImmReleaseContext (000D00CC, 0044C2B0): stub
038c:fixme:msvcp:_Locinfo__Locinfo_ctor_cat_cstr (02D1FCCC 1 C) semi-stub
038c:fixme:msvcp:_Locinfo__Locinfo_ctor_cat_cstr (02D1FB3C 1 C) semi-stub
01b0:fixme:file:errno_to_status Converting errno 75 to STATUS_UNSUCCESSFUL
01b0:fixme:file:errno_to_status Converting errno 75 to STATUS_UNSUCCESSFUL
01b0:err:virtual:virtual_setup_exception stack overflow 516 bytes in thread 01b0 addr 0x7bc56fe8 stack 0x230dfc (0x230000-0x231000-0x330000)

Running the latest version of Manjaro with Xanmod (if it matters) and Wine 6.23.

**Comment by 32th-System** [comment]
If you scroll up, does it say `C++ EH type: .?AVfilesystem_error@filesystem@std@@` somewhere?

**Comment by McUberStein** [comment]
Yes it does actually. Here: https://i.imgur.com/Dk9Kv7Z.png . I also switched to Wine-tkg-staging 7.0 to see if it would change anything, but it's still exactly the same.

**Comment by 99kaits** [comment]
having the same bug on wine-staging 7.0 from the arch repos. same exact output in terminal and on the patcher itself

**Comment by McUberStein** [comment]
I just made a Debian/Arch dual boot, so I decided to try Thcrap again to see if it would make a difference. On Arch, it's exactly the same as Manjaro, but on Debian it works perfectly. It doesn't get stuck, it doesn't crash and it works absolutely perfectly. Don't know why it doesn't work on Arch-based distros though.

**Comment by kabessao** [comment]
I read this issue a while back trying to use thcrap on my games bought from steam with no luck.

Well guess what, I finally got lucky!

I was able to patch all my games on steam using proton, and that involves no terminal usage whatsoever. 

What you need is:
* make sure you don't need to go through any dot folder to get to the game folder. One example is just creating a symlink to the .steam folder (or .local/share/Steam if .steam is not present on your home folder);
* add thcrap.exe as a "non steam game" and under properties, on compatibility, set the checkbox to select an specific proton version. Use the experimental one;
* launch thcrap.exe through steam and follow it's instructions, when prompted select "Find games in a specific folder" and navigate to the game folder (that is why we create the symlink before. The explorer on proton can't see dot folders. Also, even if you have multiple games, select the specific game folder and do the same for the other ones, thcrap doesn't detect multiple games on proton it seems);
* on "create shortcuts" you can choose whatever, just remember where you created them. I choose to create on thcrap folder;
* after patching, go back to steam and add the .lnk files to steam as "non steam games" as well, and set it to use proton experimental too, the same way you did on thcrap.exe.

And done! Everything works just as it was on windows, and when I launch the .lnk steam detects its running the original games and marks it as running on my profile.
